docs/installation/linux.md

```diff
@@ -657,19 +657,19 @@ pwsh
 
 ```sh
 # Grab the latest App Image
-wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/powershell-6.0.0-rc-x86_64.AppImage
+wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc.2/powershell-6.0.0-rc.2-x86_64.AppImage
 
 # Make executable
-chmod a+x powershell-6.0.0-rc-x86_64.AppImage
+chmod a+x powershell-6.0.0-rc.2-x86_64.AppImage
 
 # Start PowerShell
-./powershell-6.0.0-rc-x86_64.AppImage
+./powershell-6.0.0-rc.2-x86_64.AppImage
 ```
 
 ### Uninstallation - Kali
 
 ```sh
-dpkg -r powershell_6.0.0-rc-1.ubuntu.16.04_amd64.deb
+dpkg -r powershell_6.0.0-rc.2-1.ubuntu.16.04_amd64.deb
 ```
 
 ## Raspbian
```

Comment:
In the "Raspbian" section, I found 2 problems:
![image](https://user-images.githubusercontent.com/127450/34018701-072a1a8c-e0e0-11e7-873c-657aeab9c3c9.png)